lib/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@microsoft/dynamicproto-js",
   "author": "Microsoft Application Insights Team",
-  "version": "0.5.2",
+  "version": "1.0.0",
```

Comment:
Was always planning on bumping to v1.

As this removes the unused tslib peerDependency, this seems like a good time.